content/views.py

```diff
@@ -117,3 +117,103 @@ def get(self, request, content_id):
             return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : results}, status=200)
         except Content.DoesNotExist:
             return JsonResponse({'MESSAGE' : 'INVALID_CONTENT_ID'}, status=400)
+
+
+class ContentOverview(View):
+    def get(self, request, content_id):
+        try:
+            content          = Content.objects.get(id=content_id)
+
+            result = {
+                'content_id'     : content.id,
+                'title_original' : content.movieoverview_set.get().title_original,
+                'release_year'   : content.release_year,
+                'genre'          : [genre.genre.name for genre in content.contentgenre_set.all()],
+                'runtime'        : content.movieoverview_set.get().runtime,
+                'description'    : content.movieoverview_set.get().description,
+            }
+            return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : result}, status=200)
+        except Content.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_CONTENTS_ID'}, status=400)
+
+
+class ContentCast(View):
+    def get(self, request, content_id):
+        try:
+            content = Content.objects.get(id=content_id)
+
+            cast_list = [
+                {
+                    'people_id'        : people.people.id,
+                    'people_name'      : people.people.name,
+                    'role'             : people.role_name,
+                    'people_image_url' : people.people.profile_image_url,
+                } for people in content.contentpeople_set.all()
+            ]
+
+            results = {
+                'cast' : cast_list,
+            }
+
+            return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : results}, status=200)
+        except Content.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_CONTENT_ID'}, status=400)
+
+
+class WatchaContent(View):
+    def get(self, request):
+        services = ContentService.objects.get(name='watcha')
```

Comment:
변수명을 명확하게 수정해주세요! 실제로 services 변수에 담긴 객체는 Service의 instance가 아니라 ContenService 인스턴스 입니다. 지금 저만 보더라도 코드를 이해하기가 어렵습니다.
```suggestion
        content_service = ContentService.objects.get(name='watcha')
```

Reply:
반영했습니다.